.ci/updatecli/updatecli-bump-stack-version.yml

```
@@ -0,0 +1,51 @@
+# update-cli configuration for automated go updates
+---
+name: Bump ESS snapshots
+
+pipelineid: 'bump-ess-snapshot-{{ requiredEnv "BRANCH" }}'
```

The fix is required in `main` but likely we can fix it here for the backports

```suggestion
pipelineid: 'bump-ess-snapshot-{{ requiredEnv "BRANCH_NAME" }}'
```